rules/aip0142/time_field_names.go

```diff
@@ -44,13 +44,21 @@ var fieldNames = &lint.FieldRule{
 			}
 		}
 
-		// Look for timestamps that do not end in `_time`.
+		// Look for timestamps that do not end in `_time` or, if repeated, `_times`.
 		if utils.GetTypeName(f) == "google.protobuf.Timestamp" && !strings.HasSuffix(f.GetName(), "_time") {
-			return []lint.Problem{{
-				Message:    "Timestamp fields should end in `_time`.",
-				Descriptor: f,
-				Location:   locations.DescriptorName(f),
-			}}
+			if !f.IsRepeated() {
+				return []lint.Problem{{
+					Message:    "Timestamp fields should end in `_time`.",
+					Descriptor: f,
+					Location:   locations.DescriptorName(f),
+				}}
+			} else if !strings.HasSuffix(f.GetName(), "_times") {
+				return []lint.Problem{{
+					Message:    "Repeated Timestamp fields should end in `_time` or `_times`.",
+					Descriptor: f,
+					Location:   locations.DescriptorName(f),
+				}}
+			}
```

Comment:
Nit, small refactoring: If we move the (repeated && non-plural-suffix) case to a leading check, it reads a bit more explicitly and leaves the non-repeated case as the "default" behavior. This also reduces indentation of the basic case.

```suggestion
			if f.IsRepeated() && !strings.HasSuffix(f.GetName(), "_times") {
				return []lint.Problem{{
					Message:    "Repeated Timestamp fields should end in `_time` or `_times`.",
					Descriptor: f,
					Location:   locations.DescriptorName(f),
				}}
			} 
			
			return []lint.Problem{{
					Message:    "Timestamp fields should end in `_time`.",
					Descriptor: f,
					Location:   locations.DescriptorName(f),
			}}
```

Reply:
Had to make a slight change.

The suggestion would return an error in the case where we have a repeated field with a suffix of "_times", which is not intended behavior.

Reply:
Oh oops, yeah, you're right, that falls through on valid plural timestamps 🤦  Apologies! In that case I think what you had before was better, and I apologize for the churn.

Reply:
I realized my previous comment might not have been clear, but please revert back to the original logic, it was better :)

Reply:
Done!